src/Compatibility/Microsoft.DotNet.ApiCompat.Task/ValidatePackageTask.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Framework;
+using Microsoft.NET.Build.Tasks;
+using Microsoft.DotNet.ApiCompatibility.Logging;
+
+namespace Microsoft.DotNet.ApiCompat.Task
+{
+    public class ValidatePackageTask : TaskBase
+    {
+        [Required]
+        public string? PackageTargetPath { get; set; }
+
+        [Required]
+        public string? RoslynAssembliesPath { get; set; }
+
+        public string? RuntimeGraph { get; set; }
+
+        public string? NoWarn { get; set; }
+
+        public bool RunApiCompat { get; set; }
```

Comment:
When would you set this to false? Should we default it to `true`?

Reply:
This was implemented as is by Anirudh and the target that invokes this task already defaults to true. If we would like to change this behavior, let's do that in a separate change.

Reply:
Actually this was a one line change so I included it in this PR. This should have been true to begin with.

Reply:
I guess my question was more, why is it there to begin with? what is the use case scenario where you want to queue all api compat runs but don't actually run?

Reply:
That was added as part of https://github.com/dotnet/sdk/commit/d22bb74e1b73006135c000323fdab8225834b133. Unfortunately neither the PR description nor the commit message explains that. @ericstj would you know why the `RunApiCompat` was added to PackageValidation?